_search-plugins/search-pipelines/using-search-pipeline.md

```diff
@@ -130,6 +130,53 @@ The response contains only the public document, indicating that the pipeline was
 ```
 </details>
 
+You can also search on multiple indices having the same default pipeline. For example, the below alias1 has 2 indices my_index1 and my_index2 and both the indices have the same default pipeline `my_pipeline` attached to them.
```

Comment:
Change "For example, the below alias1 has 2 indices my_index1 and my_index2 and both the indices have the same default pipeline `my_pipeline` attached to them."

To "For example, below **alias1** has 2 indices **my_index1** and **my_index2** and both indices have same default pipeline `my_pipeline` attached to them."

Reply:
+1